modular/executor/migrate_task.go

```diff
@@ -139,15 +162,15 @@ func (e *ExecuteModular) doObjectMigration(ctx context.Context, task coretask.Mi
 	migratePieceTask := &gfsptask.GfSpMigratePieceTask{
```

Comment:
if Signature field is never used in GfSpMigratePieceTask, should we consider to remove it?

Reply:
Fixed, redundant fields and logic have been removed. The current proto is not used, so incompatible adjustments can be made.